Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-DM-3-B.yml

```diff
@@ -16,7 +16,7 @@ id:
 
 #The title of this activity
 title:
-  Use trend analysis to influence changes in the Design and Implementation phase across multiple projects.
+  Use trend analysis to influence changes in development across multiple projects.
```

Comment:
Maybe: "Use trend analysis to introduce proactive improvements across multiple projects"?

Reply:
OK done.